src/modules/colorPicker/ColorPickerUI/Helpers/ColorRepresentationHelper.cs

```diff
@@ -99,6 +101,29 @@ private static string ColorToHSB(Color color)
                  + $", {brightness.ToString(CultureInfo.InvariantCulture)}%)";
         }
 
+        /// <summary>
+        /// Return a <see cref="string"/> representation float color styling(0.1f, 0.1f, 0.1f)
+        /// </summary>
+        /// <param name="color">The <see cref="Color"/> to convert</param>
+        /// <returns>a string value (0.1f, 0.1f, 0.1f)</returns>
+        private static string ColorToFloat(Color color)
+        {
+            var (red, green, blue) = ColorHelper.ConvertToDouble(color);
+            var precision = 2;
+
+            return $"({Math.Round(red, precision):0.##}f, {Math.Round(green, precision):0.##}f, {Math.Round(blue, precision):0.##}f, 1f)";
```

Comment:
This still prints a format like `(0,12f, 0,12f, 0,12f, 1f)` due to my culture settings.
CIELAB for example doesn't have this issue locally. Perhaps a similar code can be used here? I think using Culture Invariant is the key here.